x-pack/test/case_api_integration/common/lib/utils.ts

```diff
@@ -577,8 +615,12 @@ export const findCasesAsUser = async ({
   return res;
 };
 
+interface OwnerEntity {
+  owner: string;
+}
+
 export const ensureSavedObjectIsAuthorized = (
-  cases: CaseResponse[],
+  cases: OwnerEntity[],
```

Comment:
Just making this more generic so we can support anything that has an `owner` field.